homeassistant/components/vicare/climate.py

```diff
@@ -134,6 +144,40 @@ def update(self):
             "circulationpump_active"
         ] = self._api.getCirculationPumpActive()
 
+        # Update the specific device attributes
+        if self._heating_type == HeatingType.gas:
+            self._attributes["burner_active"] = self._api.getBurnerActive()
+            self._attributes["burner_modulation"] = self._api.getBurnerModulation()
+            self._attributes["boiler_temperature"] = self._api.getBoilerTemperature()
+            self._attributes["current_power"] = self._api.getCurrentPower()
+            self._attributes[
+                "gas_consumption_heating_days"
```

Comment:
We prefer to make many individual sensors instead of putting measurements as many state attributes. We should only use state attributes when the measurement doesn't make sense on its own.

Reply:
I'm not sure if single sensors for all these attributes would make much sense since most users will not be interested in their heating curve slope or burner modulation.

I would however agree to move the statistics data into sensors but in that case we should do this in a separate PR.

Reply:
Just to give you an idea what data this provides here is the output of my instance:

gas_consumption_heating_days:
  - 16
  - 63
  - 24
  - 68
  - 71
  - 63
  - 55
  - 52
gas_consumption_heating_today: 16
gas_consumption_heating_weeks:
  - 181
  - 165
  - 145
  - 157
  - 101
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
  - 10
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0
  - 3
  - 89
  - 155
  - 248
  - 274
  - 326
  - 168
  - 184
  - 365
  - 287
  - 354
  - 386
  - 427
  - 408
  - 376
  - 484
  - 518
  - 593
  - 661
  - 790
  - 723
  - 649
  - 662
  - 720
  - 566
  - 735
  - 414
  - 656
  - 656
  - 488
  - 361
  - 353
  - 335
  - 212
gas_consumption_heating_this_week: 181
gas_consumption_heating_months:
  - 171
  - 1260
  - 577
  - 0
  - 10
  - 0
  - 10
  - 986
  - 1103
  - 1730
  - 2072
  - 3143
  - 2667
gas_consumption_heating_this_month: 171
gas_consumption_heating_years:
  - 11062
  - 14596
  - 16816
gas_consumption_heating_this_year: 11062

Reply:
I don't know how to fix this right now. Maybe we can approve it right now and move it to a sensor in another PR? Or I could remove the values from this PR and @oischinger can readd them later.

Reply:
Please remove them for now.

Reply:
Removing only the consumption attributes would work for me. If that works for @MartinHjelmare please go ahead.

As stated before I wouldn't want to create sensors for everything but it makes sense for the consumption stuff.

Reply:
Removed the attributes for now.